lib/service/fastethereumservice.ts

```diff
@@ -25,7 +26,7 @@ export = module.exports = class FastEthereumService extends EthereumService {
    * @param {number}   [options.interval] sync retry interval
    * @param {Logger}   [options.logger] logger instance
    */
-  constructor (options: any) {
+  constructor (options?: any) {
```

Comment:
Was this optionality of `options` necessary? I thought there is always passed an `options` dict around?

Reply:
Did this in response to a compilation error...there was an empty constructor in the tests here...
https://github.com/ethereumjs/ethereumjs-client/blob/12ea4cdec363bc848f9fdf58c9094053b94f7a83/test/service/fastethereumservice.js#L26-L29